### PR TITLE
Fixed running code and updating code for Runtime

### PR DESCRIPTION
**Pull request by AVSurfer123**

### CODE REVIEW

File: renderer/components/UpdateBox.tsx [hunk 1]
```diff
@@ -45,29 +44,32 @@ class UpdateBoxContainer extends React.Component<Props, State> {
...
+    }).then((dialogReturn: OpenDialogReturnValue) => {
+      if (dialogReturn.filePaths.length > 0) {
+        this.setState({ updateFilepath: dialogReturn.filePaths[0] });
+      }
     });
   }
 
   upgradeSoftware = () => {
     this.setState({ isUploading: true });
     const update = pathToName(this.state.updateFilepath);
```
Comment (ewc340) [inline renderer/components/UpdateBox.tsx:60]: This appears to be unused, do we still want to keep it here? If not, we can just delete it.

**Review by ewc340 — CHANGES_REQUESTED** [review]
@AVSurfer123 Just 1 question.

File: renderer/components/UpdateBox.tsx (hunk 1 above)
Comment (AVSurfer123) [reply renderer/components/UpdateBox.tsx]: Yea, I'll delete

**Review by ewc340 — APPROVED** [review]
LGTM

**Pull request merged**